[rocBLAS] default hipblaslt batched on (#1189)
ROCm/rocm-libraries: projects/rocblas/docs/reference/env-variables.rst

        | **0**: Tensile is always used as the GEMM backend.
        | **1**: hipBLASLt is preferred as the GEMM backend, but will fallback to Tensile on problems for which hipBLASLt does not provide a solution or when errors are encountered using the hipBLASLt backend.

    * - | ``ROCBLAS_USE_HIPBLASLT_BATCHED``

>         | Manual control to selectively disable use of hipblaslt only with regards to the batched GEMMs. ``ROCBLAS_USE_HIPBLASLT=0`` disables the ``ROCBLAS_USE_HIPBLASLT_BATCHED`` variable, as hipblaslt would not be enabled.

Reviewer: ```suggestion
        | Manual control to selectively disable the use of hipBlasLt only for the batched GEMMs. ``ROCBLAS_USE_HIPBLASLT=0`` disables the ``ROCBLAS_USE_HIPBLASLT_BATCHED`` variable, because hipBlasLt would not be enabled.
```

      - 1
      - :ref:`Control the GEMM backend <rocblas-tensile-hipblaslt>`
      - | **Unset**: GEMM batched default backend.

>         | **0**: Tensile will always used as the GEMM batched backend.

Reviewer 2: add the word   `be`    as below
Tensile will always `be` used as the GEMM batched backend.

>         | **1**: hipBLASLt when applicable will be used as the GEMM batched backend, but will fallback to Tensile on problems for which hipBLASLt does not provide a solution or when errors are encountered using the hipBLASLt backend.

Reviewer: ```suggestion
        | **1**: hipBLASLt will be used as the GEMM batched backend when applicable, but will fallback to Tensile on problems for which hipBLASLt does not provide a solution or when errors are encountered using the hipBLASLt backend.
```
